tutorial/08-configure-app-only.md

```diff
@@ -0,0 +1,72 @@
+---
+ms.localizationpriority: medium
+---
+
+<!-- markdownlint-disable MD041 -->
+
+In this section you will update the app registration from the previous section to support app-only authentication. You only need to complete this section if you intend to use the app-only portions of this tutorial. If not, you can safely skip to the next step.
```

Comment:
I wanted to add app only because we get pretty regular feedback about not covering how to do it in our tutorials. I thought about doing it as a separate tutorial, but this way the user has a single test harness they can drop in any code to, including APIs that only support app-only.

Reply:
Let's maybe add a sentence on why. I agree it should be there, let's just push a bit more on the why

Reply:
Let me know what you think of the addition.